### PR TITLE
update firststeps.md (fixes #625)

### LINKED ISSUE #625: Firsts Steps New Interns Progess Tracking
### Problem
To make sure you can keep a track of progress we can add the progress links to the relevant steps on the way, steps 3 & 6

 ### Steps to reproduce the problem
Go to https://treehouses.io/#!pages/vi/firststeps.md
See Step 3 and 6

 ### Screenshots
Drag and drop images below. Write N/A if not applicable.
image

 ### Proposed solution
Would add 
https://treehouses.io/#!./pages/vi/trackprogress.md to steps 3 & 6 similar to the format in 8 to track progress

### PR DESCRIPTION
**Pull request by rohansuri17**
Fixes #625 

### Description
Added a link to check progress in steps 3 & 6 so you aren't lost when you reach step 8 
### Raw.Githack preview link
https://raw.githack.com/rohansuri17/rohansuri17-OLE.github.io/Fix-Progess/pages/vi/firststeps.md

e.g. https://raw.githack.com/UserName/Repository/Branch/Path/To/File.md

### CODE REVIEW

**Comment by law-lin** [comment]
For future reference, add a #! before the pages in https://raw.githack.com/rohansuri17/rohansuri17-OLE.github.io/Fix-Progess/pages/vi/firststeps.md, so that we can view the Markdown rendering, like this:

https://raw.githack.com/rohansuri17/rohansuri17-OLE.github.io/Fix-Progess/#!pages/vi/firststeps.md

**Review by law-lin — DISMISSED** [review]
Instead of writing out the entire link, let's give a title for the link like this:

image
This is [an example](http://example.com/ "Title") inline link.

**Comment by dogi** [comment]
> Instead of writing out the entire link, let's give a title for the link like this:
> 
> image
> This is [an example](http://example.com/) inline link.

also please do not call the whole url if shorter options like relative ones or just filenames are working

**Comment by rohansuri17** [comment]
Fixed let me know if anything else needs to be done.

File: pages/vi/firststeps.md [hunk 1]
```diff
@@ -114,6 +114,8 @@ Make sure that you've linked to your github.io and pull request in the [Gitter c
 
 * 1 Pull request made
 
+Check your progress [here](https://treehouses.io/#!./pages/vi/trackprogress.md)
```
Comment (dogi) [inline pages/vi/firststeps.md]: please do not call the whole url if shorter options like relative ones or just filenames are working

File: pages/vi/firststeps.md [hunk 2]
```diff
@@ -145,6 +147,8 @@ Make sure you have created at least one issue, resolved it, commented on an issu
 * 1 comment added (on an issue you didn't create)
 * 1 issue created
 
+Check your progress [here](https://treehouses.io/#!./pages/vi/trackprogress.md)
```
Comment (dogi) [inline pages/vi/firststeps.md]: please do not call the whole url if shorter options like relative ones or just filenames are working

Comment (rohansuri17) [reply pages/vi/firststeps.md]: Sorry about that. I fixed it.

**Review by dogi — APPROVED** [review]

**Pull request merged**